### DISCUSSION TITLE
How to add margin bottom in web chatwoot.

### DISCUSSION THREAD

**Discussion by paulovitorba** [chatwoot/chatwoot#7713] [Q&A] [answered]
I've tried several options that I know of and I can't add the bottom margin, can someone from the community help me please?
https://prnt.sc/OzyEWlrxXiiH
https://prnt.sc/6fB0BHEMUAB9

**Comment by mmojadad** [comment, marked as answer]
Hello
I had same problem and solved it via my website theme custom css, I share with you and hope to be useful.
In my opinion chatwoot must have such feature in built-ed for example define custom font family,size ... etc for widget.
/* text */
#woot-widget--expanded__text {
	padding-left:15px;
	margin-right: -30px;
}
/* icon */
#woot-widget-bubble-icon {
	padding-right:15px;
}

**Reply by paulovitorba** [reply]
I added the following code to my HTML

        <style>
            .woot-widget-bubble {
        
          bottom: 120px !important;
        
          z-index: 1 !important;
            }
          .chatwoot-container   {
            margin-bottom: 79px;
            }
        </style>

      <div class="chatwoot-container">
        <script>
          window.chatwootSettings = {"position":"right","type":"standard","launcherTitle":"Fale conosco no chat"};
          (function(d,t) {
            var BASE_URL="https://dominio.com";
            var g=d.createElement(t),s=d.getElementsByTagName(t)[0];
            g.src=BASE_URL+"/packs/js/sdk.js";
            g.defer = true;
            g.async = true;
            s.parentNode.insertBefore(g,s);
            g.onload=function(){
              window.chatwootSDK.run({
                websiteToken: '*****************************',
                baseUrl: BASE_URL
              })
            }
          })(document,"script");
        </script>
      </div>

Working fine, now I will test your option, Thanks for the help.

**Comment by iiihexiii** [comment]
how to reach it
